src/views/open-api-docs-view/server.ts

```diff
@@ -71,7 +68,7 @@ export async function getStaticProps({
 	serviceProductSlug = productSlug,
 	versionData,
 	basePath,
-	statusIndicatorConfig,
+	statusIndicatorConfig = null, // returned in getStaticProps, avoid undefined
```

Comment:
Bit of unrelated cleanup here, figured worth sneaking in 👍

Reply:
"must be JSON-serializable" would be a good distinction to note!